makefile for fibonacci proof erroring on master branch

Currently, running ```make fibonacci-proof.json``` on master causes the following error: 

```
➜  examples git:(master) ✗ make fibonacci-proof
cargo run --release -- prove --expression fibonacci.lurk --proof fibonacci-proof.json
    Finished release [optimized] target(s) in 0.09s
     Running `/Users/jonat/lurk-rs/target/release/fcomm prove --expression fibonacci.lurk --proof fibonacci-proof.json`
thread 'main' panicked at 'failed to read file: Error("expected value", line: 1, column: 1)', /Users/jonat/lurk-rs/fcomm/src/lib.rs:413:44
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
make: *** [fibonacci-proof.json] Error 101
```

README instructions are [here](https://github.com/lurk-lang/lurk-rs/tree/master/fcomm#creating-and-verifying-evaluation-proofs).

The above command _does_ work on the fcomm-commitment branch. On master, you have to run the cargo command with --lurk added to make it work (```cargo run --release -- prove --expression fibonacci.lurk --proof fibonacci-proof.json --lurk```). 

It seems like either the JSON needs to be fixed to work as it does on fcomm-commitment branch or the makefile on master needs to update to include --lurk.

--- a/fcomm/examples/Makefile
+++ b/fcomm/examples/Makefile
@@ -5,10 +5,10 @@ eval-fibonacci :
 	cargo run --release -- eval --expression fibonacci.lurk
 
 fibonacci-claim.json : fibonacci.lurk
-	cargo run --release -- eval --expression fibonacci.lurk --claim fibonacci-claim.json
+	cargo run --release -- eval --expression fibonacci.lurk --claim fibonacci-claim.json --lurk
 
 fibonacci-proof.json : fibonacci.lurk
-	cargo run --release -- prove --expression fibonacci.lurk --proof fibonacci-proof.json
+	cargo run --release -- prove --expression fibonacci.lurk --proof fibonacci-proof.json --lurk
 
 fibonacci-proof2.json : fibonacci-claim.json
 	cargo run --release -- prove --claim fibonacci-claim.json --proof fibonacci-proof2.json

